tests/rkt_tests.go

```diff
@@ -133,19 +135,28 @@ func (ctx *rktRunCtx) dir(idx int) string {
 }
 
 func (ctx *rktRunCtx) reset() {
+	ctx.cleanupChildren()
 	ctx.runGC()
 	for _, d := range ctx.directories {
 		d.reset()
 	}
 }
 
+func (ctx *rktRunCtx) cleanupChildren() {
+	for _, child := range ctx.children {
+		log.Printf("Shutting down %q", child.Cmd.Path)
+		child.Cmd.Process.Kill()
```

Comment:
I suggest doing a nominal check and just logging for information purposes. I'm just trying to future-proof us against the kind of murky mad situation that got us here in the first place.

If you feel strongly against that, let's at least move to the better practice/style of explicitly ignoring the error:
`_ = child.Cmd.Process.Wait()`